docs/source/index.rst

```diff
@@ -71,8 +71,9 @@
    installation/index
    changes
    contributing
-   archive
+   component_docs
```

Comment:
Use `components` instead of `component_docs`? (Also affects dynamic generation above and .gitignore).

Reply:
Done